persistence/persistence-api/src/main/java/org/cbioportal/persistence/StructuralVariantRepository.java

```diff
@@ -0,0 +1,11 @@
+package org.cbioportal.persistence;
+
+import org.cbioportal.model.StructuralVariant;
+
+import java.util.List;
+
+public interface StructuralVariantRepository {
+
+    List<StructuralVariant> fetchStructuralVariants(List<String> molecularProfileIds, 
```

Comment:
oh i see here... so the SV dao is only for importing and the SV repository is only for fetching then, which is why the dao doesn't have a fetcher method. If so then disregard my earlier comment concerning the fetcher method in the dao.

Reply:
yes, that's it. I am going to resolve the other comments about this topic 😃